Common/include/option_structure.hpp

```diff
@@ -625,13 +625,15 @@ enum ENUM_LIMITER {
   VENKATAKRISHNAN = 0,	/*!< \brief Slope limiter using Venkatakrisnan method. */
   BARTH_JESPERSEN = 1,  /*!< \brief Slope limiter using Barth-Jespersen method. */
   SHARP_EDGES = 2,       /*!< \brief Slope limiter using sharp edges. */
-  SOLID_WALL_DISTANCE = 3       /*!< \brief Slope limiter using wall distance. */
+  SOLID_WALL_DISTANCE = 3,       /*!< \brief Slope limiter using wall distance. */
+	VAN_ALBADA = 4         /*!< \brief Slope limiter using wall distance. */
```

Comment:
I have a bunch of test-cases in which the Van Albada works much better than the Venkatakrishnan. Usually what happens is that the residuals stall with the Venka, but instead
they keep going down with the Van Albada.